addons/website_sale/views/templates.xml

```diff
@@ -1212,5 +1212,211 @@
+                            </t>
+                          </div>
+                        </t>
+                      </div>
+                    </t>
+                  </div>
+                  <div id="shipping_address" class="col-md-6">
+                    <div>
+                      <strong>Shipping Address</strong>
+                    </div>
+                    <div>
+                      <address t-field="o.partner_shipping_id" t-field-options='{"widget": "contact", "fields": ["address", "name"], "no_marker": true}'/>
+                    </div>
+                  </div>
+                </div>
+
+                <hr/>
+
+                <div id="product_lines">
+                  <!-- Product lines come here -->
+                </div>
+
+                <hr/>
+
+                <div class="row">
+                  <div class="col-md-6">
+                    <div>
+                      <strong>Contact</strong>
+                    </div>
+                    <div t-field="o.user_id.partner_id" t-field-options='{"widget": "contact", "fields": ["email", "phone"]}'/>
+                  </div>
+                  <div class="col-md-6">
+                    <div class="row">
+                      <div class="col-md-10 text-right">
+                        Untaxed Amount:
+                      </div>
+                      <div class="col-md-2 text-right">
+                        <span t-field="o.amount_untaxed" t-field-options='{"widget": "monetary", "display_currency": "o.company_id.currency_id"}'/>
+                      </div>
+                    </div>
+                    <div class="row">
+                      <div class="col-md-10 text-right">
+                        Taxes:
+                      </div>
+                      <div class="col-md-2 text-right">
+                        <span t-field="o.amount_tax" t-field-options='{"widget": "monetary", "display_currency": "o.company_id.currency_id"}'/>
+                      </div>
+                    </div>
+                    <div class="row">
+                      <div class="col-md-10 text-right">
+                        <strong>Total:</strong>
+                      </div>
+                      <div class="col-md-2 text-right">
+                        <strong><span t-field="o.amount_total" t-field-options='{"widget": "monetary", "display_currency": "o.company_id.currency_id"}'/></strong>
+                      </div>
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </t>
+
+        <t t-if="len(orders) >= 2">
+          <t t-call="website_sale.orders_followup_navbar"/>
+        </t>
+        <h4 class="text-muted text-center mt64 mb64">
+          If you wish to access the complete application to manage your orders, you can follow 
+          <a href="/shop/backend">this link</a>.
+        </h4>
```

Comment:
Special last minute request from @antonylesuisse : put a link that can lead customers to the backend if they wish (i.e. for big clients with hundreds of records)